dotcom-rendering/docs/testing.md

```diff
@@ -75,32 +75,14 @@ The first time you run Playwright you will be asked to install a local browser,
 
 `pnpm playwright install chromium`
 
-Running Playwright locally requires having a DCR server running.
-
-#### Against the PROD server
+Running Playwright locally will automatically start the dev server.
 
 To run the tests on the command line in headless mode:
 
-`make playwright`
+`make playwright-run`
 
-To run the tests in the Playwright UI:
+To open the tests in the Playwright UI:
 
 `make playwright-open`
```

Comment:
Running Playwright locally is much simpler now just either:

`make playwright-run`

or 

`make playwright-open`